docs/ref/checks.txt

```diff
@@ -127,9 +127,11 @@ MySQL and MariaDB
 If you're using MySQL or MariaDB, the following checks will be performed:
 
 * **mysql.E001**: MySQL/MariaDB does not allow unique ``CharField``\s to have a
-  ``max_length`` > 255.
+  ``max_length`` > 255. *This check appeared before Django 3.1.*
```

Comment:
Couldn't this be a versionchanged annotation? Also oudl clarify a bit more that it was changed to W003

Reply:
Added.